content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md

```diff
@@ -1426,6 +1426,127 @@ Here's what each section of the example code does:
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-19.png) 
+
+Add one variable with the following characteristics:
+
+- **Name**: `led`
+- **Variable type**: `boolean`
+- **Variable permission** `Read & Write`
+- **Variable update policy**: `On change`
+
+You should see the `led` variable on the **Cloud Variables** section. Now, navigate into **Dashboards** and select the **BUILD DASHBOARD** button, this will create a new dashboard; give your dashboard a name. Now, add the following to your widgets to your dashboard:
+
+- **Switch**: name the widget `Switch` and link it to the `led` variable you created before. 
+- **LED**: name the widget `led` and link it to the `led` variable you created before. 
+
+Your dashboard should see like the following:
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-20.png)
+
+Now, go back to your **Things** and open the "Thing" you just have created. In the "Thing" setup page, navigate into **Sketch**, you should see now an online editor. 
+
+In the generated sketch, you just need to define the `led` pin as an output:
+
+```arduino
+// LED_BUILTIN macro access the onboard green LED 
+pinMode(LED_BUILTIN, OUTPUT);  
+```
+
+In the `onLedChange()` function, you just need to set the LED state with the `led` variable.
+
+```arduino
+// led variable synced with the Arduino IoT Cloud dashboard 
+digitalWrite(LED_BUILTIN, led);  
+```
+
+The full example code can be found below: 
+
+```arduino
+/* 
+  Sketch generated by the Arduino IoT Cloud
+
+  The following variables are automatically generated and updated when changes are made to the Thing
+
+  bool led;
+*/
+
+#include "thingProperties.h"
+
+void setup() {
+  // Initialize serial and wait for port to open:
+  Serial.begin(9600);
+
+  // This delay gives the chance to wait for a Serial Monitor without blocking if none is found
+  delay(1500); 
+
+  // LED_BUILTIN macro access the onboard green LED 
+  pinMode(LED_BUILTIN, OUTPUT);
+
+  // Defined in thingProperties.h
+  initProperties();
+
+  // Connect to Arduino IoT Cloud
+  ArduinoCloud.begin(ArduinoIoTPreferredConnection);
+  
+  /*
+     The following function allows you to obtain more information
+     related to the state of network and IoT Cloud connection and errors
+     the higher number the more granular information you’ll get.
+     The default is 0 (only errors).
+     Maximum is 4
+ */
+  setDebugMessageLevel(2);
+  ArduinoCloud.printDebugInfo();
+}
+
+void loop() {
+  ArduinoCloud.update();
+}
+
+
+/*
+  Since Led is READ_WRITE variable, onLedChange() is
+  executed every time a new value is received from IoT Cloud.
+*/
+void onLedChange()  {
+  // led variable synced with the Arduino IoT Cloud dashboard 
+  digitalWrite(LED_BUILTIN,led);
+}
+```
+

```

Comment:
We cannot finish the section this way

We have to finish with:
* Explain the actions the user need to do after it (compile & upload)
* Explain the user what are the final results of these actions to verify everything is working fine (as we said in our last meeting) in this case, explaining that once the code is uploaded to the board, the user should be capable of controlling the LED thought the dashboard and see if it is on or off in real-time.

Reply:
IMPORTANT

C33 Led is OFF when siwthc is ON and vice-versa, we need to change the code to make the example coherent!!!! @canchebagur

Reply:
I didn't found another interesting widget we can use... Any idea?

Reply:
Ok let's do one thing, let keep it this way while I look for alternatives but please do:

"We cannot finish the section this way

We have to finish with:

* Explain the actions the user need to do after it (compile & upload)
* Explain the user what are the final results of these actions to verify everything is working fine (as we said in our last meeting) in this case, explaining that once the code is uploaded to the board, the user should be capable of controlling the LED thought the dashboard and see if it is on or off in real-time.
"

Thanks!!!

Reply:
Ok, a screenshot and a small section explaining the compile and upload process were added. Also, a small section explaining to the users the final results of those actions.